src/main/java/com/keshavarzi/airplanestats/exception/register/AuthorizationRoleMissingException.java

```diff
@@ -0,0 +1,15 @@
+package com.keshavarzi.airplanestats.exception.register;
+
+public class AuthorizationRoleMissingException extends Exception {
+    /**
+     * Constructs a new exception with the specified detail message.  The
+     * cause is not initialized, and may subsequently be initialized by
+     * a call to {@link #initCause}.
+     *
+     * @param message the detail message. The detail message is saved for
+     *                later retrieval by the {@link #getMessage()} method.
+     */
+    public AuthorizationRoleMissingException(String message) {
+        super("Security role " + message + " was not found.");
+    }
```

Comment:
Nit: Generally, exception strings should not embed into other strings. Either have not wrapping text or place the input text after.